moped-editor/src/views/projects/signalProjectTable/SignalProjectTable.js

```diff
@@ -287,15 +287,19 @@ const SignalProjectTable = () => {
       title: "Project DO#",
       field: "purchase_order_number",
       emptyValue: "-",
+      render: entry =>
+        entry.purchase_order_number.trim().length === 0
+          ? "-"
+          : entry.purchase_order_number,
```

Comment:
this allows blank project DO's to still be editable

Reply:
✂️  `trim()` FTW!